SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -556,6 +570,30 @@ class CVariable {
       val_residual[iVar] = Residual_Sum(iPoint,iVar);
   }
 
+  /*!
+   * \brief Set the value of the under-relaxation parameter for the current control volume (CV).
+   * \param[in] val_under_relaxation - the input value of the under-relaxation parameter for this CV.
+   */
+  inline void SetUnderRelaxation(unsigned long iPoint, su2double val_under_relaxation) { UnderRelaxation(iPoint) = val_under_relaxation; }
+  
+  /*!
+   * \brief Get the value of the under-relaxation parameter for the current control volume (CV).
+   * \return Value of the under-relaxation parameter for this CV.
+   */
+  inline su2double GetUnderRelaxation(unsigned long iPoint) { return UnderRelaxation(iPoint); }
+  
+  /*!
+   * \brief Set the value of the local CFL number for the current control volume (CV).
+   * \param[in] val_cfl - the input value of the local CFL number for this CV.
+   */
+  inline void SetLocalCFL(unsigned long iPoint, su2double val_cfl) { LocalCFL(iPoint) = val_cfl; }
+  
+  /*!
+   * \brief Get the value of the local CFL number for the current control volume (CV).
+   * \return Value of the local CFL number for this CV.
+   */
+  inline su2double GetLocalCFL(unsigned long iPoint) { return LocalCFL(iPoint); }
```

Comment:
Thanks for adopting the `const` and `final` keywords for the gradient stuff, these two "gets" can be `const` too.

Reply:
Done